language/it/permissions_ajax_shoutbox.php

```diff
@@ -45,6 +45,5 @@
 		'ACL_U_SHOUTBOX_BBCODE' => 'Può usare BBCode in bacheca',
 		'ACL_U_SHOUTBOX_DELETE' => 'Può rimuovere i propri messaggi dalla bacheca',
 		'ACL_M_SHOUTBOX_DELETE' => 'Può rimuovere messaggi dalla bacheca',
-		'ACL_M_SHOUTBOX_EDIT'   => 'Può modificare messaggi in bacheca',
```

Comment:
Hello @paul999,
why has this permission been removed at last?
Shouldn't moderators be able to edit other users' messages?